EIPS/eip-X-ASIC-resistant-block-mining.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: <to be assigned>
+title: Poisoning the Well
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: <email address>
+status: Draft
+type: Standards Track
+category Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP attempts to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and probabalistically actively mining.  This EIP aims to "Poison
+the well" by modifying the block mining algorithm in a manner that
+probabalistically *"breaks"* these miners if they are in-fact built on ASICs.
+
+
+## Motivation
+
+ASIC based miners will have lower operational costs than GPU based miners which
+will result in GPU based mining quickly becoming unprofitable.  Given that
+production of ASIC based miners has a high barrier to entry, this will cause a
```

Comment:
Production of GPUs also has a high barrier to entry.  The provided argument does not effectively explain why ASICs are bad and GPUs are good.  Recommend refining this to provide a more clear argument for why ASICs will lead to centralization.

Reply:
" more clear argument for why ASICs will lead to centralization" in every instance that an ASIC has been produced, it has led to a greater degree of centralization.  no need to create an argument on something that is common sense these days... this is not pre-bitcoin ASIC days

Reply:
Data showing a statistically significant correlation and causation between introduction of ASICs into a crypto-currency and an increase in centralization.  I have seen a lot of correlation claims, but none backed up by data that gives a reasonable argument for causation other than "it feels right".  When it comes to reality, there are often things that _feel_ correct but turn out not to be.

Keep in mind that miners do not have a meaningful vote in which direction a hard fork goes.  Economic participants make the decisions as to which fork survives and thrives, it only takes a single miner (until PoS, when it takes 0 miners) in alignment with economic participants to keep a fork alive.